2025-04-09 00:50:20 - bot_launcher - INFO - ╟ряєёъ Telegram сюЄр
2025-04-09 00:50:20 - bot_launcher - INFO - ╤яЁюср чряєёъє сюЄр #1
2025-04-09 00:50:20 - bot_launcher - INFO - ╟ряєёър║ью Telegram сюЄр...
2025-04-09 00:50:20 - telegram_bot - INFO - Django setup successful
2025-04-09 00:50:24 - telegram_bot - ERROR - Critical error in bot initialization: name 'router' is not defined
2025-04-09 00:50:24 - telegram_bot - ERROR - Traceback (most recent call last):
  File "C:\Users\domik\OneDrive\╨рсюўшщ ёЄюы\parsing_group_tg\tg_bot\bot.py", line 31, in <module>
    from tg_bot.handlers import common_router, admin_router, session_router
  File "C:\Users\domik\OneDrive\╨рсюўшщ ёЄюы\parsing_group_tg\tg_bot\handlers\__init__.py", line 2, in <module>
    from tg_bot.handlers.admin import admin_router
  File "C:\Users\domik\OneDrive\╨рсюўшщ ёЄюы\parsing_group_tg\tg_bot\handlers\admin.py", line 100, in <module>
    @router.message(F.text == "\U0001f4ce List of channels", F.from_user.id == ADMIN_ID)
     ^^^^^^
NameError: name 'router' is not defined

2025-04-09 00:50:24 - bot_launcher - WARNING - ═х тфрыюё  │ьяюЁЄєтрЄш сюЄр эряЁ ьє, тшъюЁшёЄютє║ью ры№ЄхЁэрЄштэшщ ьхЄюф
2025-04-09 00:50:24 - bot_launcher - INFO - ╟ряєёър║ью сюЄр ўхЁхч я│фяЁюЎхё
2025-04-09 00:50:30 - bot_launcher - INFO - ┴юЄ єёя│°эю чряє∙хэю ўхЁхч я│фяЁюЎхё
2025-04-09 00:50:30 - bot_launcher - INFO - ┴юЄ єёя│°эю чряє∙хэю
